File: adt2data.c
//  This file is part of Adlib Tracker II (AT2).
//
// SPDX-FileType: SOURCE
// SPDX-FileCopyrightText: 2014-2024 The Adlib Tracker 2 Authors
// SPDX-License-Identifier: GPL-3.0-or-later

#include "defines.h"

#include "adt2data.h"

unsigned char font8x16[4096] = {
#include "font/track16.inc"
};

unsigned char vga_font8x16[4096] = {
#include "font/vga16.inc"
};

#if !GO32

unsigned char adt2_icon_bitmap[3126] = {
#include "adtrack2-icon.inc"
};

#endif // !GO32

File: adt2data.pas
//  This file is part of Adlib Tracker II (AT2).
//
//  AT2 is free software: you can redistribute it and/or modify
//  it under the terms of the GNU General Public License as published by
//  the Free Software Foundation, either version 3 of the License, or
//  (at your option) any later version.
//
//  AT2 is distributed in the hope that it will be useful,
//  but WITHOUT ANY WARRANTY; without even the implied warranty of
//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
//  GNU General Public License for more details.
//
//  You should have received a copy of the GNU General Public License
//  along with AT2.  If not, see <http://www.gnu.org/licenses/>.

unit AdT2data;
{$S-,Q-,R-,V-,B-,X+}
{$PACKRECORDS 1}
{$L adt2data.o}
interface

var
  font8x16: array[0..4095] of Byte; cvar; external;
  vga_font8x16: array[0..4095] of Byte; cvar; external;
{$IFNDEF GO32V2}
  adt2_icon_bitmap: array[0..3125] of Byte; cvar; external;
{$ENDIF}

implementation

end.
